devnet/docker-compose-substrate.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  prometheus:
+    image: prom/prometheus
+    volumes:
+    - ./../prometheus.yml:/etc/prometheus/prometheus.yml
+    command:
+    - '--config.file=/etc/prometheus/prometheus.yml'
+    - '--storage.tsdb.path=/prometheus'
+    - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+    - '--web.console.templates=/usr/share/prometheus/consoles'
+    ports:
+    - 9090:9090
+    restart: always
+
+  alice:
+    platform: linux/amd64
+    build:
+      context: ./..
+      dockerfile: devnet/substrate_alice.Dockerfile
+    ports:
+      - 30333
+      - 9933
+      - 9944
+      - 9615:9615
```

Comment:
```suggestion
      - 9615
```

Reply:
can we make these exposed ports the same as gossamer? Or we should at least comment what the diff ports do.

Reply:
Yes, it is possible!